server/types/slack.d.ts

```diff
@@ -0,0 +1 @@
+declare module "@slack/client";
```

Comment:
this is necessary when a third-party library does not either:

1) ship it's own types in it's own package
2) publish community-maintained types in the DefintelyType repo https://github.com/DefinitelyTyped/DefinitelyTyped